mmv1/third_party/terraform/tpgresource/import.go

```diff
@@ -64,18 +65,46 @@ func ParseImportId(idRegexes []string, d TerraformResourceData, config *transpor
 			}
 
 			// The first id format is applied first and contains all the fields.
-			err := setDefaultValues(idRegexes[0], d, config)
+			err := setDefaultValues(idRegexes[0], nil, d, config)
 			if err != nil {
 				return err
 			}
 
 			return nil
+		} else if d.Id() == "" {
```

Comment:
We can move `if d.Id() == "" ` before line 30 to first check if d.Id()  is empty?

Reply:
this made sense to do, i've applied it in the latest commit